packages/compass-connections/src/components/connections.tsx

```diff
@@ -83,30 +86,37 @@ function Connections({
     isConnected,
   } = state;
 
+  // For now the connections sidebar is always dark
+  const [theme] = useState<ThemeState>({
+    theme: Theme.Dark,
+  });
+
   return (
     <div
       data-testid={
         isConnected ? 'connections-connected' : 'connections-disconnected'
       }
       className={connectStyles}
     >
-      <ResizableSidebar darkMode>
-        <ConnectionList
-          activeConnectionId={activeConnectionId}
-          favoriteConnections={favoriteConnections}
-          recentConnections={recentConnections}
-          createNewConnection={createNewConnection}
-          setActiveConnectionId={setActiveConnectionById}
-          onDoubleClick={(connectionInfo) => {
-            void connect(connectionInfo);
-          }}
-          removeAllRecentsConnections={() => {
-            void removeAllRecentsConnections();
-          }}
-          removeConnection={removeConnection}
-          duplicateConnection={duplicateConnection}
-        />
-      </ResizableSidebar>
+      <ThemeProvider theme={theme}>
```

Comment:
I very like this pattern of using provider to force / invert the theme where needed, means that you can safely change anything lower in the react tree and don't need to always keep in mind which parts of compass are inverted / always dark mode, etc. Really good stuff!